app/helpers/races_helper.rb

```diff
@@ -0,0 +1,19 @@
+module RacesHelper
+  def statcard_class(race)
+    case
+    when !race.started?
+      'text-success'
+    when race.in_progress?
+      'text-warning'
+    when race.finished?
+      nil
+    else
+      raise 'invalid status'
+    end
+  end
+
+  # Overrides the default helper because Race#to_param returns a user-friendly ID
+  def api_v4_race_path(race)
+    super(race.id)
+  end
```

Comment:
this felt super dirty to do, open to better ideas if you have them

Reply:
The straightforward answer would be to either have the API take friendly URLs as well, or just forgo them completely.

Reply:
yeah, but
1. i hate super long and unintelligible user-facing urls
2. i regret ever making api resources accessible by anything but id (like games and runners) and want to move away from it long-term

so i think this might be the best of bad options :/